src/server/search/search_family_test.cc

```diff
@@ -570,7 +570,7 @@ TEST_F(SearchFamilyTest, SimpleExpiry) {
   EXPECT_THAT(Run({"ft.search", "i1", "*"}), AreDocIds("d:1"));
 
   Run({"flushall"});
-  EXPECT_EQ(used_mem_current.load(), 0);
```

Comment:
Apparently the heartbeat has enough time on CI to recompute the memory budget

Reply:
I don't understand this comment :upside_down_face: 
And also - why do you now drop the index? Don't you want at least to verify we don't get an error reply?
I guess `FLUSHALL` shouldn't drop indices, but I just want to make sure that's how it's designed to work?

Reply:
> I don't understand this comment 🙃

used_mem_current is always zero, unless the heartbeat does more than 8 iterations and the value is updated. Because the we preallocate some stuff, it goes up immediately. I added this check because it blocked other values from being added

Reply:
And also - why do you now drop the index?

It slipped in accidentally when I experimented with this value and I just didn't exclude it... Just removed it

Reply:
I don't think we should be dropping indices on `flushall`, but just wanted to make sure you agree :)
Anyway, sg.

Reply:
we don't drop